### PR TITLE
Add script to upload replies and tags to firebase

### PR DESCRIPTION
**Pull request by marianamarasoiu**
This is a bit hacky but works for now.

@lukechurch I've also added a couple of test/sample files for how the input should look like. Let me know if it looks ok, or if you'd prefer a different format.

Thanks!

### CODE REVIEW

File: tool/tests/test_tags.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      "shortcut": "o",
+      "type": "normal"
+    },
+    {
+      "text": "STOP",
+      "shortcut": "",
+      "type": "important"
+    },
+    {
+      "text": "STOP",
```
Comment (danrubel) [inline tool/tests/test_tags.json]: Just checking that this is an intended duplicate of the entry above

Comment (marianamarasoiu) [reply tool/tests/test_tags.json]: It wasn't intentional, thanks for the catch!

File: tool/add_to_firebase.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+def init(CRYPTO_TOKEN_PATH):
+    global firebase_client
+    log.info("Setting up Firebase client")
+    firebase_cred = credentials.Certificate(CRYPTO_TOKEN_PATH)
+    firebase_admin.initialize_app(firebase_cred)
+    firebase_client = firestore.client()
+    log.info("Done")
+
+def push_collection_to_firestore(collection_root, documents):
```
Comment (lukechurch) [inline tool/add_to_firebase.py]: Please add issue to refactor out to helper library

File: tool/add_to_firebase.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+if CONTENT_TYPE not in valid_content_type:
+    log.error("Content type '{}' not known, expected one of {}".format(CONTENT_TYPE, valid_content_type))
+    exit(1)
+
+with open(INPUT_PATH, 'r') as f:
+    data_dict = json.load(f)
+
+if CONTENT_TYPE == "suggested_replies":
+    # Make sure that the fields that the Firestore model expects exist
+    assert len(data_dict.keys()) == 1
```
Comment (lukechurch) [inline tool/add_to_firebase.py]: Move to validator in the fullness of time

File: tool/add_to_firebase.py (hunk 2 above)
Comment (marianamarasoiu) [reply tool/add_to_firebase.py]: issue created, thanks

File: tool/add_to_firebase.py (hunk 3 above)
Comment (marianamarasoiu) [reply tool/add_to_firebase.py]: Thanks, I've added a couple of comments to keep track of this and created an issue too.

**Comment by marianamarasoiu** [comment]
Thanks, merging now..

**Pull request merged**